Description: Fix pointer type
 tmp_pcrs is being allocated a memory with sizeof(int) and value obtained
 from atoi() is stored in it. Since the value stored and memory allocated
 are both 'int', the pointer should be 'int *'.

Author: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Bug-Debian: https://bugs.debian.org/1074931
Last-Update: 2025-02-27
---


--- ecryptfs-utils-111.orig/src/utils/ecryptfs_generate_tpm_key.c
+++ ecryptfs-utils-111/src/utils/ecryptfs_generate_tpm_key.c
@@ -89,7 +89,7 @@ int main(int argc, char **argv)
 	int i, c, *pcrsSelected = NULL, numPcrsSelected = 0;
 	TSS_UUID *uuid;
 	BYTE wellknown[] = TSS_WELL_KNOWN_SECRET;
-	char *tmp_pcrs;
+	int *tmp_pcrs;
 
 	while (1) {
 		c = getopt(argc, argv, "p:");
